drivers/ethernet/eth_mcux.c

```diff
@@ -1021,7 +1022,11 @@ static void eth_mcux_init(const struct device *dev)
 	context->phy_handle->ops = &phyksz8081_ops;
 
 #if defined(CONFIG_SOC_SERIES_IMX_RT10XX)
+#if defined(CONFIG_BOARD_MIMXRT1062_FMURT6)
+	sys_clock = CLOCK_GetFreq(kCLOCK_EnetPll1Clk);
```

Comment:
I'm a bit concerned about introducing this board specific change here. Why does the FMURT board need a different source clock?

Reply:
Such is the board design.
ETH_REF_CLK line between MAC and PHY has only ENET2 in its Pinmux option, such is the board design.

Reply:
Made similar changes here by adding DT_NODE_HAS_STATUS(DT_NODELABEL(enet), okay) and removed than board specific MACRO